conf/sources.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
We had migrated `sources` configure to each cluster. We should remove this file.

Reply:
done!